packages/dds/tree/src/simple-tree/toMapTree.ts

```diff
@@ -290,13 +312,55 @@ function objectToMapTree(data: InsertableContent, schema: TreeNodeSchema): MapTr
 	// Filter keys to only those that are strings - our trees do not support symbol or numeric property keys
 	const keys = Reflect.ownKeys(data).filter((key) => typeof key === "string") as FieldKey[];
 
+	const allKeys = Object.keys((schema as any).info) as FieldKey[];
+	const keysWithoutData = allKeys.filter((value) => !keys.includes(value));
+	for (const key of keysWithoutData) {
```

Comment:
I think it would be simpler and more efficient to just loop like:
```suggestion
	for (const [key, fieldSchema] of Object.entries(schema.info as Record<string, ImplicitFieldSchema>) {
	   if (!keys.includes(key)) {
	      ...
```

That avoids two copies of the key list, and the lookup of the field schema.
Then to handle implicit field schema,
```typescript
if (fieldSchema instanceof FieldSchema) {
// Only FieldSchema instances can have defaults
const defaultProvider = fieldSchema.props?.defaultProvider;
...
```

Reply:
Updated to more efficient loop, and used instanceof for typecheck